contracts/OpenSTValue.sol

```diff
@@ -256,9 +259,12 @@ contract OpenSTValue is OpsManaged, Hasher {
 
         emit ProcessedStake(stake.uuid, _stakingIntentHash, stakeAddress, stake.staker,
             stake.amountST, stake.amountUT, _unlockSecret);
+        /** Remove intent hash from intents mapping */
```

Comment:
correct comment type, and insert whiteline above; and to be inline with your style in `revertStaking` remove the whiteline between the deletes